draft-ietf-httpbis-client-hints.md

```diff
@@ -169,6 +200,17 @@ Implementers ought to consider both user and server controlled mechanisms and po
 Implementers SHOULD support Client Hints opt-in mechanisms and MUST clear persisted opt-in preferences when any one of site data, browsing history, browsing cache, or similar, are cleared.
 
 
+## Deployment and Security Risks
+Deployment of new request headers requires several considerations:
+
+  - Potential conflicts due to existing use of field name
+  - Properties of the data communicated in field value
+
+Specifications and features relying on Client Hints are advised to carfully consider prefixing request header names with a "Sec-" prefix. User agents, such as web browsers, ensure such prefixed headers can only be emitted by the browser, and application content, such as scripts, are forbidden from emitting such headers {{FETCH}}. This provides assurance that, when transmitted by user agents, the type, format and value of data communicated in such request headers are enforced by the user agent.
```

Comment:
carfully?

Also, the number, of, commas, here, is, starting to, get tiresome.  Maybe there are two sentences in that second one.

Reply:
I think it's much more clear to say something like:

Authors of new Client Hints are advised to carefully consider whether they should be able to be added by client-side content (e.g., scripts), or whether they should be exclusively set by the user agent. In the latter case, the `Sec-` prefix on the header field name has the effect of preventing scripts and other application content from setting them in many user agents. See {{FETCH}} for more information.

Reply:
That is definitely better phrasing.  Assuming that you still want to do this based on [my earlier comments](https://github.com/httpwg/http-extensions/pull/776#discussion_r342875878).

Reply:
Oops regarding "carfully"...

Otherwise, I like @mnot's proposed text, assuming we are fine with user-added Client-Hints. I need to give them some more thought.

Reply:
Yeah, Mark's text seems fine to me.

Reply:
> Using the "Sec-" prefix signals to servers that the user agent - and not application content - generated the values.